samples/unit-tests/3d/column-animation/demo.js

```diff
@@ -1,6 +1,8 @@
 /* eslint func-style:0 */
 
-QUnit.test('Point animation', function (assert) {
+// Skipped, can't seem to find out why this fails after SVG path refactoring.
```

Comment:
Maybe it is a timing problem?

Reply:
Turns out it's a real problem with path animation. It jumps to the end path before it starts animating. Thanks for noticing, I will work on a fix.

Reply:
Fixed

Reply:
That was fast! 😄